functions/system.bash

```diff
@@ -69,6 +69,8 @@ timezone_setting() {
 
 locale_setting() {
   cond_redirect apt-get -q -y install locales
+  # shellcheck disable=SC1090
+  source "$CONFIGFILE" 
```

Comment:
see problem with init.bash above, consider hard coding the filename

Reply:
ok, I added configfile= to /etc/openhabian.conf. Sort of self referential but it is not a config option and I dislike hardcoding strings in many places